src/main/java/de/dennisguse/opentracks/util/ExceptionHandler.java

```diff
@@ -0,0 +1,90 @@
+package de.dennisguse.opentracks.util;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+
+import androidx.annotation.NonNull;
+
+import java.util.Arrays;
+
+import de.dennisguse.opentracks.BuildConfig;
+import de.dennisguse.opentracks.ShowErrorActivity;
+
+public class ExceptionHandler implements Thread.UncaughtExceptionHandler {
+
+    private static final int EXCEPTION_FORMAT_MAX_RECURSIVITY = 10;
```

Comment:
Why 10? And not say: 3?

Reply:
That was the value defined in the Nextcloud code. I didn't question it. Do you think 3 is enough?

Reply:
Gone